evalml/automl/automl_search.py

```diff
@@ -681,6 +686,52 @@ def _compute_cv_scores(self, pipeline, X, y):
         logger.info(f"\tFinished cross validation - mean {self.objective.name}: {cv_score_mean:.3f}")
         return {'cv_data': cv_data, 'training_time': training_time, 'cv_scores': cv_scores, 'cv_score_mean': cv_score_mean}
 
+    def _compute_ensemble_scores(self, pipeline, X, y):
+        start = time.time()
+        cv_data = []
+        X.reset_index(drop=True, inplace=True)
```

Comment:
I'm still trying to figure out why this line is necessary. Works without locally, but breaks when running perf tests without. 🤔

Reply:
edit: testing a few things, maybe this won't be necessary